src/utils/data-management/downloadFromUrl.js

```diff
@@ -1,8 +1,10 @@
-const downloadFromUrl = (url) => {
+const downloadFromUrl = (url, filename) => {
   const link = document.createElement('a');
   link.style.display = 'none';
   link.href = url;
 
+  if (filename) link.download = filename;
```

Comment:
If it is an optional parameter, I would then have `filename` in the function parameters like this: `(..., filename = null) => {...}` so that it is more obvious.

Reply:
Also perhaps check with `isNil`?

Reply:
I don't know if anyone will input the filename as `false` or `0`, but ok, I think it's a valid point.